README.md

```diff
@@ -106,35 +113,45 @@ An example of a configuration file can be found in [test_config.ini](https://git
 
 After processing the configuration file, DaLI reads crypto data from native sources and generates a RP2 input ODS file and a RP2 configuration file in the `output` directory or where specified with the `-o` CLI option.
 
-To try DaLI with the example configuration, download the following files:
-* [test_config.ini](https://github.com/eprbell/dali-rp2/tree/main/config/test_config.ini)
-* [test_manual_in.csv](https://github.com/eprbell/dali-rp2/tree/main/input/test_manual_in.csv)
-* [test_manual_intra.csv](https://github.com/eprbell/dali-rp2/tree/main/input/test_manual_intra.csv)
-* [test_trezor_alice.csv](https://github.com/eprbell/dali-rp2/tree/main/input/test_trezor_alice_btc.csv)
-* [test_trezor_bob.csv](https://github.com/eprbell/dali-rp2/tree/main/input/test_trezor_bob_btc.csv)
+## Quickstart
```

Comment:
I would remove the "Quickstart" title, but not a strong opinion. If you leave it:
* make it a ### title
* update the Table of Contents